docs/content/docs/development/write-table.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Write Table"
+weight: 3
+type: docs
+aliases:
+- /development/write-table.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Write Table
+
+```sql
+INSERT { INTO | OVERWRITE } [catalog_name.][db_name.]table_name
+  [PARTITION part_spec] [column_list] select_statement
+
+part_spec:
+  (part_col_name1=val1 [, part_col_name2=val2, ...])
+
+column_list:
+  (col_name1 [, column_name2, ...])
+```
+
+## Unify Streaming and Batch
```

Comment:
Let's spend more effort on polishing the user story. The current expression is a bit colloquial, not enough written and formal.

Reply:
How about =>


Flink Table Store supports read/write under both batch and streaming mode. Beyond that, it can also write to the same managed table simultaneously by different streaming and batch tasks.

Suppose you have a partitioned table defined as
```sql
-- a managed table DDL
CREATE TABLE MyTable (
  user_id BIGINT,
  item_id BIGINT,
  dt STRING
) PARTITIONED BY (dt);
```
And there is a real-time pipeline to perform the data sync task, followed by the downstream jobs to perform the rest ETL steps.
```sql
-- Run a streaming job that continuously writes to the table
SET 'execution.runtime-mode' = 'streaming';
INSERT INTO MyDwdTable SELECT user_id, item_id FROM MyCdcTable WHERE some_filter;
INSERT INTO MyDwsTable 
SELECT dt, item_id, COUNT(user_id) FROM MyDwdTable GROUP BY dt, item_id;
```
Some backfill tasks are often required to correct historical data, which means you can start a new batch job overwriting the table's historical partition without influencing the current streaming pipeline and the downstream tasks.
```sql
-- Run a batch job to revise yesterday's partition
SET 'execution.runtime-mode' = 'batch';
INSERT OVERWRITE MyDwdTable PARTITION ('dt'='20220402')
SELECT user_id, item_id FROM MyCdcTable WHERE dt = '20220402' AND new_filter;
```

Reply:
And maybe we can better illustrate this use case with some pic.
![image](https://user-images.githubusercontent.com/55568005/161418656-302a9d53-dfa1-43cc-ab62-25b3b81aebf5.png)